README.md

```diff
@@ -0,0 +1,201 @@
+# ROLE REPO MIRROR
+
+[![image](https://img.shields.io/github/license/adfinis-sygroup/ansible-role-repo_mirror.svg?style=flat-square)](https://github.com/adfinis-sygroup/ansible-role-repo_mirror/blob/master/LICENSE)
+
+[![image](https://img.shields.io/travis/adfinis-sygroup/ansible-role-repo_mirror.svg?style=flat-square)](https://github.com/adfinis-sygroup/ansible-role-repo_mirror)
+
+[![image](https://img.shields.io/badge/galaxy-adfinis--sygroup.repo_mirror-660198.svg?style=flat-square)](https://galaxy.ansible.com/adfinis-sygroup/repo_mirror)
+
+Ansible role that helps deploying a package mirror
+
+## Requirements
+
+-   A debian system with bullseye is required
+-   systemd as init system is required
+-   Before you use this role, create a storage pool with something like
+    ZFS and mount it at the location where the mirrored repositories
+    would appear.
+
+## Role Variables
+
+```yaml
+# vars/Debian.yml
```

Comment:
What's all of this doing in `vars/Debian.yml`? Shouldn't most of these be set in defaults and be overridden in hostvars?

Reply:
That was a mistake while copy/pasting

Reply:
The packages id like to keep in the vars file tho.